src/os/windows/crashlog_win.cpp

```diff
@@ -403,11 +392,11 @@ static INT_PTR CALLBACK CrashDialogFunc(HWND wnd, UINT msg, WPARAM wParam, LPARA
 			}
 			*p = '\0';
 
-			/* Add path to crash.log and crash.dmp (if any) to the crash window text */
+			/* Add path to all files to the crash window text */
 			size_t len = wcslen(_crash_desc) + 2;
 			len += wcslen(convert_to_fs(CrashLogWindows::current->crashlog_filename, filenamebuf, lengthof(filenamebuf))) + 2;
 			len += wcslen(convert_to_fs(CrashLogWindows::current->crashdump_filename, filenamebuf, lengthof(filenamebuf))) + 2;
-			len += wcslen(convert_to_fs(CrashLogWindows::current->screenshot_filename, filenamebuf, lengthof(filenamebuf))) + 1;
+			len += wcslen(convert_to_fs(CrashLogWindows::current->screenshot_filename, filenamebuf, lengthof(filenamebuf))) + 2;
 
 			static wchar_t text[lengthof(_crash_desc) + 3 * MAX_PATH * 2 + 7];
```

Comment:
```suggestion
			static wchar_t text[lengthof(_crash_desc) + 3 * MAX_PATH * 2 + 8];
```
to match the change in `len`calculation I think. (Though we already double the paths max length).

Reply:
I misunderstood what the `* 2` was doing .. it is to reserve space for the wchar .. (UTF-16 and all) .. oops. Reverting my change here :)